### ISSUE TITLE
Panic when loading track

### ISSUE THREAD

**Issue by tmplt** [Spotifyd/spotifyd#897]
**Description**
spotifyd v0.3.0 panics when I play a song.

**To Reproduce**
1. `nix run -f channel:nixpkgs-unstable spotifyd`.
2. Write into `~/.config/spotifyd/spotifyd.conf`:
    ```toml
    [global]
    username = "[REDACTED]"
    password = "[REDACTED]"
    backend = "pulseaudio"
    device_name = "perscitia"
    bitrate = 320
    no_audio_cache = true
    device_type = "computer"
    ```

3. Start spotifyd via `spotifyd --no-daemon`.
4. Select spotifyd via Connect and play a track in a playlist.
5. Observe panic.

**Logs**
```
Loading config from "/home/tmplt/.config/spotifyd/spotifyd.conf"
No proxy specified
Using software volume controller.
Connecting to AP "gew1-accesspoint-e-bhhz.ap.spotify.com:443"
Authenticated as "[REDACTED]" !
Country: "SE"
Loading <Nuclear> with Spotify URI <spotify:track:754WgEesWgssZG3XzU9XcG>
Caught panic with message: attempted to zero-initialize type `librespot_tremor::tremor_sys::ov_callbacks`, which is invalid
Caught panic with message: called `Result::unwrap()` on an `Err` value: "SendError(..)"
```

`--verbose` spits out a lot of info. I have yet to go through it to redact any eventual credentials.

**Versions (please complete the following information):**

- OS: NixOS
- Spotifyd: v0.3.0
- cargo: N/A

**Comment by tmplt** [comment]
Probably the same issue as in #719.

**Comment by emanuelserpa** [comment]
same thing on arch using the spotifyd-full-git package.

**Comment by robinvd** [comment]
This should be fixed in 0.3.2 what version are you using @emanuelserpa `spotifyd --version`

**Comment by emanuelserpa** [comment]
Nevermind, was my mistake. I was using spotifyd as a nix package and forgot to update.

**Comment by robinvd** [comment]
I closed it, but if these is still an issue on 0.3.2 please reopen @tmplt

**Comment by tmplt** [comment]
Resolved on 0.3.2 on my end. Thanks.
